docs/rfc/rfc-1.adoc

```diff
@@ -346,19 +346,15 @@ Governable Parameters:
 - `wallet_min_btc`: The smallest amount of btc a wallet can hold before we
   attempt to close the wallet and transfer the funds to a randomly selected
   wallet.
-- `wallet_max_btc`: The amount of btc a wallet can hold before we trigger the
-  early creation of a new wallet.
 - `wallet_max_age`: The oldest we allow a wallet to become before we transfer the funds
   to a randomly selected wallet.
 
 A new wallet is created when either enough time has passed as defined in
-`wallet_creation_period` *AND* the wallet contains at least
-`wallet_min_btc` btc *OR* the wallet currently contains more btc than
-`wallet_max_btc`, whichever happens first. To create a new wallet, a group of
-100 operators is selected from the pool of available operators using a process
-called sortition. The probabiliy that a particular operator is chosen is based
-on their stake weight, which in turn is based on the number of `T` tokens they
-have invested in the staking contract.
+`wallet_creation_period` *AND* the wallet contains at least `wallet_min_btc`
```

Comment:
```suggestion
`wallet_creation_period` *OR* the wallet contains at least `wallet_min_btc`
```

Reply:
This actually is correct at "and". We only want to make a new wallet if the current one has >= `wallet_min_btc` after a week, otherwise, we wait until next week and repeat!

Reply:
This is actually correct at *AND*. We create a new wallet every ~week (the `wallet_creation_period`), but only if the wallet has >= `wallet_min_btc`. If it doesn't, we skip wallet creation that week and repeat. Maybe next week it will have accumulated some more BTC and we will create a new wallet when we check again.

Reply:
Oh right, I see now that `AND` should indeed be there. In that case I think we should remove `either` in line 354/352.

Reply:
@michalinacienciala Please see https://github.com/keep-network/tbtc-v2/pull/77/commits/31d5d3402c8ddf946aea18944f1d825f6239a879